abstractions/php/src/RequestInfo.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Microsoft\Kiota\Abstractions;
+
+use Psr\Http\Message\StreamInterface;
+
+class RequestInfo {
+    public ?string $uri;
+    public ?HttpMethod $httpMethod;
+    public StreamInterface $contents;
+    public array $headers = [];
+    public array $queryParams = [];
+    private static string $binaryContentType = "application/octet-stream";
+    private static string $contentTypeHeader = "Content-Type";
```

Comment:
FYI this is missing the set content from parsable method which I've reordered parameters in #480 to support varargs and raw collections.